src/components/Button.js

```diff
@@ -1,15 +1,14 @@
-import { React } from "react";
-import PropTypes from "prop-types";
+import { React } from 'react';
+import PropTypes from 'prop-types';
 
-export const Button = (props) => {
+export const Button = props => {
```

Comment:
- [ ] **[OPTIONAL]** We can directly de-structure objects inside function arguments, which will save use a few lines.

```js
export const Button = ({ name, color, wide}) => {
```